pages/common/yt-dlp.md

```diff
@@ -9,11 +9,11 @@
 
 `yt-dlp "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
 
-- List the available downloadable formats for a video:
+- List all formats that a video or playlist is available in:
 
 `yt-dlp --list-formats "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
 
-- Download a video with a defined format, in this case the best mp4 video available (default is "bv\*+ba/b"):
+- Download a video or playlist at a specific quality:
```

Comment:
```suggestion
- Download a video or playlist using best MP4 video available (default is "bv\*+ba/b"):
```
What about this? I agree with @CleanMachine1, your change removes key information for the user.